.circleci/config.yml

```diff
@@ -154,7 +154,7 @@ jobs:
           condition: << parameters.lint >>
           steps:
             - run: find . -path '*.podspec' -exec perl -pi -e 's/.+\.social_media_url.+//' {} \;
-            - run: bundle exec pod lib lint MapboxCoreNavigation.podspec
+            - run: bundle exec pod lib lint MapboxCoreNavigation.podspec --allow-warnings
```

Comment:
Temporary workaround. Will be removed in future.

Reply:
Let’s ticket out removing this workaround as tail work and link to the ticket as a known issue in release notes.

Reply:
Yep, created ticket in: https://github.com/mapbox/mapbox-navigation-ios/issues/2951.